javacutil/src/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -170,6 +170,14 @@ private static DeclaredType typeFromName(Elements elements, CharSequence name) {
     }
 
     /**
+     * @return the simple name of an annotation as a String
+     */
+    public static String annotationSimpleName(AnnotationMirror annotation) {
+        String annotationName = annotationName(annotation);
+        return annotationName.substring(annotationName.lastIndexOf('.') + 1 /* +1 to skip the last . as well */);
```

Comment:
Do you have a feel how much this method will be used?
If there is a cache miss in annotationName, the fully-qualified name will be computed and cached and then here you strip off the fully-qualified part.
If there is a cache hit, you still always have to strip off the fully-qualified part.
It might be nicer to have a separate cache for the simple names, implement it like annotationName, but call https://docs.oracle.com/javase/7/docs/api/javax/lang/model/element/TypeElement.html#getSimpleName() instead of getQualifiedName().